batch2/batch/batch.py

```diff
@@ -114,41 +114,28 @@
     await notify_batch_job_complete(db, batch_id)
 
 
-def job_record_to_dict(record):
-    def getopt(obj, attr):
-        if obj is None:
-            return None
-        return obj.get(attr)
+def job_record_to_dict(record, running_status=None):
+    spec = json.loads(record['spec'])
+
+    attributes = spec.pop('attributes', None)
 
     result = {
         'batch_id': record['batch_id'],
         'job_id': record['job_id'],
-        'state': record['state']
+        'state': record['state'],
+        'spec': spec
```

Comment:
this will include the docker image in the CI UI page, right? Awesome!

Reply:
Yes!